example/lib/main.dart

```diff
@@ -56,6 +56,9 @@ class _MyAppState extends State<MyApp> {
                   width: 150,
                   height: 30,
                   unsupportedPlatformChild: DownloadPass(pkPass: _pkPassData),
+                  onPress: () {
```

Comment:
Flutter is calling `onPressed` this action in their buttons. Should we go with that `standard` way if we think to make this public?